src/commodore/inventory.spec.ts

```diff
@@ -0,0 +1,41 @@
+import yaml from 'js-yaml';
+import { mkdir, readFile, unlink } from 'fs/promises';
+
+import { loadFixture } from '../test/util';
+import { expect, describe, it } from '@jest/globals';
+
+import * as inv from './inventory';
+import { cacheDir } from './util';
+import { Facts } from './types';
+
+beforeAll(async () => {
+  await mkdir(cacheDir(), { recursive: true });
+});
+
+describe('src/commodore/inventory', () => {
+  describe('writeFactsFile()', () => {
+    it('writes facts to file', async () => {
+      const f = new Facts();
+      f.distribution = 'dist';
+      f.cloud = 'cloud';
+      f.region = 'region';
+      const factsPath = await inv.writeFactsFile('ck', f);
+
+      expect(factsPath).toBe(`${cacheDir()}/ck-facts.yaml`);
+
+      const resStr = (await readFile(factsPath)).toString();
+      const res = yaml.load(resStr);
```

Comment:
```suggestion
      const res: any = yaml.load(resStr) as any;
```
Again my LSP will complain, as `res` has type `unknown` and you technically cannot access any fields of it.